.github/workflows/ci.yml

```diff
@@ -30,6 +30,7 @@ jobs:
         run: |
           eval $(opam env)
           opam update
+          opam install dune
```

Comment:
That is really weird: why do we need that? It seems to have fixed the problem but the `opam install --deps-only -d -t bindlib` should take care of that.

Reply:
I think this is a temporary bug in opam packages ? the installation of dune via ocaml 4.08 for older version of ocaml
was failing at opam upgrade but opam install dune works ...I think it is harmless anyway ? We can recheck in a few days 
if opam is updated ? Or open an issue on opam ?

Reply:
I leave the conversation unresolved ... We will decide before you are ready to merge. Maybe everything will be fixed by that time ?

Reply:
OK, let's see.